plugins/Login/Emails/LoginFromDifferentCountryEmail.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link    https://matomo.org
+ * @license https://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\Login\Emails;
+
+use Piwik\Date;
+use Piwik\Intl\Data\Provider\DateTimeFormatProvider;
+use Piwik\Mail;
+use Piwik\Piwik;
+use Piwik\Plugin\Manager as PluginManager;
+use Piwik\Url;
+use Piwik\View;
+
+class LoginFromDifferentCountryEmail extends Mail
+{
+    /**
+     * @var string
+     */
+    private $login;
+
+    /**
+     * @var string
+     */
+    private $email;
+
+    /**
+     * @var string
+     */
+    private $country;
+
+    /**
+     * @var string
+     */
+    private $ip;
+
+
+    public function __construct($login, $email, $country, $ip)
+    {
+        parent::__construct();
+
+        $this->login = $login;
+        $this->email = $email;
+        $this->country = $country;
+        $this->ip = $ip;
+
+        $this->setUpEmail();
+    }
+
+    private function setUpEmail()
+    {
+        $this->setDefaultFromPiwik();
+        $this->addTo($this->email);
+        $this->setSubject($this->getDefaultSubject());
+        $this->addReplyTo($this->getFrom(), $this->getFromName());
+        $this->setWrappedHtmlBody($this->getDefaultBodyView());
```

Comment:
We should consider having a text only variant for this security relevant email as well.